server.js

```diff
@@ -26,6 +26,7 @@ const typeDefs = gql`
   }
   type Mutation {
     singleUpload(file: Upload!): UploadedMediaResponse!
+    coverUpload(file: Upload!): UploadedMediaResponse!
```

Comment:
I had considered simply making thumbnails out of every image that gets uploaded anyway... thoughts?

Reply:
I don't think that you'll need it for all...  Like library items or what about profile images.